### ISSUE TITLE
attachment save return error 'A facade root has not been set'

### ISSUE THREAD

**Issue by rsanche** [Webklex/php-imap#87]
Not sure if this is a bug. I can connect and retrieve emails and attachments. 
However, when I try to save the attachment using the save function, i'm getting this error:

PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'A facade root has not been set.' in /var/www/html/.../vendor/illuminate/support/Facades/Facade.php:210

I'm using php-imap standalone

Sample Code:
```
    if($message->hasAttachments())
    {
        foreach($message->getAttachments() as $attachment)
        {
            print '<pre>'.print_r($attachment->getAttributes(), true).'</pre>'; // this is working, I see the file content, the attachment name, etc
            $attachment->save(__DIR__."/reconciliation/downloaded"); // this is gitting the error 'A facade root has not been set'
        }
    }
```

Thank you

**Comment by Webklex** [comment]
Hi @rsanche ,
may thanks for your report. 

I was able to reproduce the problem and provided a fix.

Please give me a few days to release a new version (v2.4.2). I'll let you know as soon as it is available.

Best regards,

**Comment by Webklex** [comment]
A new version is available; Please checkout v2.4.2.

Best regards,
